File: MoveMoney-SPA/src/app/_models/Customer.ts
import { TypeIdentification } from './TypeIdentification';

export interface Customer {

    id: number;
    firstName: string;
    lastName: string;
    typeIdentificationId: number;
    identification: string;
    phoneNumber: string;
    dateOfBirth?: Date;
    country: string;
    city: string;
    address?: string;
}

export declare type Customers = Customer[]

export interface CustomerParams {
    customers: Customers;
}

File: API/Models/Customer.cs
using System;

namespace MoveMoney.API.Models
{
    public class Customer
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public TypeIdentification TypeIdentification { get; set; }
        public int TypeIdentificationID { get; set; }
        public string Identification { get; set; }
        public DateTime Created { get; set; }
        public string PhoneNumber { get; set; }
        public DateTime? DateOfBirth { get; set; }
        public string Country { get; set; }
        public string City { get; set; }
        public string Address { get; set; }
        public Customer()
        {
            this.Created = DateTime.Now;
        }
    }
}